Source/TrainConsole/Data.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+using System.Linq;
+
+namespace TrainConsole
+{
+    public class Data
+    {
+        public List<Track> Tracks = new List<Track>();
+        public List<TrackSwitch> TrackSwitches = new List<TrackSwitch>();
+        public List<Station> Stations = new List<Station>();
+        public List<Train> Trains = new List<Train>();
+        public List<Gate> Gates = new List<Gate>();
+        public List<Trip> Trips = new List<Trip>();
+        public List<Passenger> Passengers = new List<Passenger>();
+        public List<TimeTable> TimeTables = new List<TimeTable>();
+
+
+
+        public void LoadAllFiles()
+        {
+            string path = @"..\..\..\..\..\Data\";
+
+            LoadFile(path, "trains.txt");
+            LoadFile(path, "passengers.txt");
+            LoadFile(path, "stations.txt");
+            LoadFile(path, "timetable.txt");
+            LoadFile(path, "traintrack.txt");
+        }
+
+        public void LoadFile(string path, string fileName)
```

Comment:
Smart att göra en metod som splittar, en annan variant skulle kunna vara att göra overloadmetoder (en för varje typ av split exempelvis. Det blir lite hårdkodning med overload då varje overload är för en typ av fil, men samtidigt har ni hårdkodning i era metoder.

Ex på overlod, se i metoden vilket tecken som splittas:
public static void SplitFile(string[] dataToSplit, List<Timetable> timetable)
        {
            for (int i = 1; i < dataToSplit.Length; i++)
            {
                string[] tmp = dataToSplit[i].Split(',');
                timetable.Add(new Timetable(tmp[3], tmp[2], tmp[0], int.Parse(tmp[1])));
            }
        }

        public static void SplitFile(string[] dataToSplit, List<Passenger> passenger)
        {
            for (int i = 1; i < dataToSplit.Length; i++)
            {
                string[] tmp = dataToSplit[i].Split(';');
                passenger.Add(new Passenger(int.Parse(tmp[0]), tmp[1]));
            }
        }

        public static void SplitFile(string[] dataToSplit, List<Train> train)
        {
            for (int i = 1; i < dataToSplit.Length; i++)
            {
                string[] tmp = dataToSplit[i].Split(',');
                train.Add(new Train(int.Parse(tmp[0]), tmp[1], int.Parse(tmp[2]), bool.Parse(tmp[3])));
            }
        }